### PR TITLE
Update media-sound/cmus to 2.9.1 from 2.7.1

### PR DESCRIPTION
**Pull request by Archetypic**

### CODE REVIEW

**Comment by heirecka** [comment]
There are few new dependencies/newer required versions:

- systemd
- bass
- wavpack >= 4.40

And it looks like building against libcue was dropped.

**Comment by Archetypic** [comment]
Which package provides `bass.h`?

**Comment by heirecka** [comment]
> Which package provides `bass.h`?

Probably https://www.un4seen.com/bass.html, but before writing an exheres for that you can just hard-disable it, to make it sure it doesn't get picked up in case we gain the package.

File: packages/media-sound/cmus/cmus.exlib [hunk 1]
```diff
@@ -77,8 +81,8 @@ DEFAULT_SRC_CONFIGURE_PARAMS=(
 DEFAULT_SRC_COMPILE_PARAMS=( V=2 ) # Maximum verbosity
 
 SRC_CONFIGURE_OPTIONS=(
-    aac alsa ao cue ffmpeg flac jack "libsamplerate samplerate" mikmod modplug
-    "mp3 mad" mp4 "musepack mpc" opus oss "pulseaudio pulse" vorbis wav wavpack
+    aac alsa ao "elogind mpris" ffmpeg flac jack "libsamplerate samplerate" mikmod modplug
```
Comment (heirecka) [inline packages/media-sound/cmus/cmus.exlib:84]: elogind and system aren't options, but suboptions with a "providers" prefix in this case.

File: packages/media-sound/cmus/cmus.exlib [hunk 2]
```diff
@@ -105,6 +106,11 @@ cmus_src_configure() {
     local s
     local myconf=(
         $(for s in "${SRC_CONFIGURE_OPTIONS[@]}"; do option_yesno ${s}; done)
+        $(if option providers:elogind || option providers:systemd; then
```
Comment (heirecka) [inline packages/media-sound/cmus/cmus.exlib:109]: You don't need this if branch, you can still pass "systemd mpris" (via `option_yesno`) to `option`, you just need to add the `providers:` prefix before the option name.

**Comment by Archetypic** [comment]
I greatly appreciate your patience.

**Pull request merged**